src/controllers/TeacherController.js

```diff
@@ -0,0 +1,48 @@
+const Teacher = require('../models').Teacher;
+const { addNew } = require('./PaymentController');
+const { deleteById } = require('./StreamController');
+
+module.exports = {
+  async getAll(req, res) {
+    try {
+      const teachers = await Teacher.findAll();
+      return res.status(200).send(teachers);
+    } catch (error) {
+      res.status(500).send(error);
+    }
+  },
+  async addNew(req, res) {
+    try {
+      const newTeacher = await Teacher.create(req.body);
+      return res.status(200).send(newTeacher);
+    } catch (error) {
+      res.status(400).send(error);
+    }
+  },
+  async updateById(req, res) {
+    try {
+      const teacher = await Teacher.findByPk(req.params.id);
+      if (teacher) {
+        const updatedTeacher = await Teacher.update(req.body);
```

Comment:
Функция редактирования данных не работает.
```suggestion
        const updatedTeacher = await teacher.update(req.body);
```

Reply:
исправлен